packages/@cdktf/provider-generator/lib/config.ts

```diff
@@ -3,7 +3,7 @@ import * as path from "path";
 import { Language } from "./get/constructs-maker";
 import { env } from "process";
 import { CONTEXT_ENV } from "cdktf";
-import { isRegistryModule } from "@cdktf/hcl2cdk";
+import { isRegistryModule } from "./get/module";
 
 const CONFIG_FILE = "cdktf.json";
 const DEFAULTS = {
```

Comment:
As Github does not allow commenting on moving files I comment here:
I'm a bit unsure about moving `readConfig` to `@cdktf/provider-generator` – as `readConfig` concerns the cdktf config. For example:
```
export interface Config {
  readonly app?: string;
  readonly language?: Language;
  readonly output: string;
  readonly codeMakerOutput: string;
  terraformProviders?: TerraformProviderConstraint[];
  terraformModules?: TerraformModuleConstraint[];
  checkCodeMakerOutput?: boolean;
  readonly context?: { [key: string]: any };
}
```

I can understand that parsing the config is required here and a circular dependency would be worse – but I'm still unsure about the naming of this package (which I like for what it does) and having things like `readonly app?: string;` in here. No blocker for me, just wanted to surface this.

Reply:
Ah, just saw your "Move the config parts into a separate package?" follow up task :D

Reply:
Yeah, this is meant to unblock our two workstreams, not as the final package architecture I'd envision for the generation parts